src/Microsoft.VisualStudio.VsInteractiveWindow/source.extension.vsixmanifest

```diff
@@ -12,15 +12,6 @@
     <InstallationTarget Version="[14.0,]" Id="Microsoft.VisualStudio.VWDExpress" />
```

Comment:
Its a few lines up, but <ShortcutPath> isn't valid anymore. You want to define the following in your csproj instead
```
<IsProductComponent>true</IsProductComponent>
<ExtensionInstallationRoot>CommonExtensions</ExtensionInstallationRoot> 
<ExtensionInstallationFolder>Microsoft\VisualStudioInteractiveWindow</ExtensionInstallationFolder> 
```

Additionally, you need a new element in your vsixmanifest here:
```
<PackageId>Microsoft.VisualStudio.InteractiveWindow</PackageId>
```